scripts/params.yaml

```diff
@@ -0,0 +1,10 @@
+gnss_host: '172.31.0.90'
+gnss_port: 3003
+ntrip_host: 'uk.nrtk.eu'
+ntrip_port: 9301
+mountpoint: 'MSM_iMAX'
+username: '40017819B'
+password: 'UOFE002'
```

Comment:
These should not be uploaded in a git commit, instead they should be a git secret.

Reply:
We will need a Github Token configured in the server. I am okay with this

Reply:
Does that mean we need internet access every time we run the script? Or are secrets somehow cloned locally?
In any case this repo needs internet to work, so it's ok in either case

Reply:
We need internet :)